src/index.ts

```diff
@@ -53,6 +53,7 @@ interface IConfig extends IStaticContext {
     customHeaders?: Record<string, string>;
     impressionDataAll?: boolean;
     usePOSTrequests?: boolean;
+    togglesStorageTTL?: number;
```

Comment:
Instead of adding it directly to the config definition, If we could have an optional "experimental" container we could make it clear that this option might go away or be changed in the future. 

```
experimental?: {
  togglesStorageTTL?: number;
}
```

Reply:
Ok we are working on it and adding the hashed context. Do you have any proposal for how you would like this to be done?

Reply:
Hello

Concerning the hashed context, we started with the use of this type of library : “[object-hash](https://www.npmjs.com/package/object-hash)”. 
Is this OK for you ?

“[unleash-proxy-client](https://bundlephobia.com/package/unleash-proxy-client@3.4.0)” lib size => 5.5 KB 
“[object-hash](https://bundlephobia.com/package/object-hash@3.0.0)” lib size => 10 KB

Which is a significant increase in percentage terms.

Last lib update: 02/2022 ([v3.0.0](https://bundlephobia.com/package/object-hash@3.0.0)), 

Otherwise we can simply use a JSON.stringify (more verbose but which does not depend on anything)

Reply:
We do not want to take on a library of this size for this. 

I think the JSON.stringify is a good start, and as this is an experimental feature we can use that to learn the performance impact here. In the future we can consider to take advantage of the built in hasing capabilities, supported by most modern browsers (https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest)